coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/FuzzyUnionTest.java

```diff
@@ -35,7 +32,7 @@ private SqlNode getFuzzyUnionView(String databaseName, String viewName) {
   }
 
   @Test
-  public void testNoSchemaEvolution() throws TException {
+  public void testNoSchemaEvolution() {
```

Comment:
I am just curious, did you catch these by running some linter tools ?

Reply:
I used the IntelliJ code analyzer 😂

Reply:
that's pretty useful static analysis ... should use it more often